po/ja.po

```diff
@@ -13132,66 +13186,88 @@ msgid ""
 "arguments passed to a function, so `entry.S` doesn't need to do anything "
 "special except make sure it doesn't change these registers."
 msgstr ""
+"`_x0`–`_x3`はレジスタ`x0`–`x3`の値であり、慣習的にブートロードがデバイスツ"
+"リーなどへのポインタを渡すのに利用されています。（`extern \"C\"`により指定さ"
+"れた）aarch64 の関数コール規約ではレジスタ`x0`–`x7`は最初の８個の引数を関数に"
+"渡すのに利用されることになっているため、`entry.S` はこれらの値を変更しないよ"
+"うにする以外の特別なことをする必要はありません。"
 
 #: src/bare-metal/aps/inline-assembly.md:56
 msgid ""
 "Run the example in QEMU with `make qemu_psci` under `src/bare-metal/aps/"
 "examples`."
 msgstr ""
+"この例を`src/bare-metal/aps/examples`において`make qemu_psci`とすることでQEMU"
+"により実行してみましょう。"
 
 #: src/bare-metal/aps/mmio.md:1
 msgid "Volatile memory access for MMIO"
-msgstr ""
+msgstr "MMIOに対するvolatileアクセス"
 
 #: src/bare-metal/aps/mmio.md:3
 msgid "Use `pointer::read_volatile` and `pointer::write_volatile`."
-msgstr ""
+msgstr "`pointer::read_volatile`と`pointer::write_volatile`を使います。"
 
 #: src/bare-metal/aps/mmio.md:4
 msgid "Never hold a reference."
-msgstr ""
+msgstr "絶対に参照を保持してはいけません。"
 
 #: src/bare-metal/aps/mmio.md:5
 msgid ""
 "`addr_of!` lets you get fields of structs without creating an intermediate "
 "reference."
 msgstr ""
+"`addr_of!`を用いると、中間的な参照を作らずに構造体のフィールドにアクセスする"
+"ことができます。"
 
 #: src/bare-metal/aps/mmio.md:9
 msgid ""
 "Volatile access: read or write operations may have side-effects, so prevent "
 "the compiler or hardware from reordering, duplicating or eliding them."
 msgstr ""
+"Volatileアクセス：（MMIO領域に対する）リードやライトは副作用があることがある"
```

Comment:
```suggestion
"Volatileアクセス：MMIO領域に対する読み書きは副作用があることがある"
```

* I think you translate "read" and "write" to "読み" and "書き" at places, respectively. Either set is fine but it should be consistent
* I don't think `()` is needed to add phrases that are omitted in the English version for most cases because normal readers don't  care about the correspondence between Japanese version and English version.

Reply:
Or, I think we can use `read` and `write` as is like "MMIO領域に対するreadやwriteは副作用があることがある". I personally feel that "read" is more natural than "リード", as it's clear that it's not "lead"